src/components/SideMenu/SideMenuContainer.js

```diff
@@ -24,52 +22,35 @@ export const mapDispatchToProps = dispatch =>
   bindActionCreators(
     {
       fetchSports: fetchSports,
-      fetchLeagues: fetchLeagues,
-      setSportsTimestamp: setSportsTimestamp,
-      setLeaguesTimestamp: setLeaguesTimestamp
+      fetchLeagues: fetchLeagues
     },
     dispatch
   );
 
 export const fetchData = props => {
-  const {
-    fetchSports,
-    categoryId,
-    fetchLeagues,
-    setSportsTimestamp,
-    setLeaguesTimestamp
-  } = props;
-  const timestamp = generateTimestamp();
-  setSportsTimestamp(timestamp);
-  setLeaguesTimestamp(timestamp);
-  fetchSports(timestamp);
-  categoryId && fetchLeagues(categoryId, timestamp);
+  const { fetchSports, categoryId, fetchLeagues } = props;
+  fetchSports();
+  categoryId && fetchLeagues(categoryId);
 };
 
 export const lifecycles = () => {
-  let timer = null;
   return {
+    timer: null,
```

Comment:
rather strange decision. do it like
`
let timer = null;

export const lifecycles = () => { ...
...
`

Reply:
And make the lifecycles to be the object. Update the test afterward

Reply:
I did it for unit testing purposes to have access to the timer property. If there is another way of accessing this variable, I would be grateful to know how to do it